asserts/ifacedecls_test.go

```diff
@@ -41,7 +41,7 @@ type attrConstraintsSuite struct {
 	testutil.BaseTest
 }
 
-func attrs(yml string) map[string]interface{} {
+func attrs(yml string) *snap.PlugInfo {
```

Comment:
I find the name of the function un-intuitive now. Can it be simply called `plug` now?
In addition this looks like something that should take `c *C` and use `c.Assert(err, IsNil)` rather than panic.

Reply:
related to this comment and the one below,  I think we if have using PlugInfo mesleading, then this tests should use a local test only implementation of Atter,  PlugInfo is only used because it's convenient but all the package cares about is the Attrer interface

Reply:
Done.